assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convert.py

```diff
@@ -37,21 +37,21 @@ def _prepare_artifacts_dict(input_dir: Path) -> Dict:
 
 
 def _get_mlflow_signature(task_type: str) -> ModelSignature:
-    """Return mlflow model signature with input and output schema given the input task type.
+    """Return MLflow model signature with input and output schema given the input task type.
```

Comment:
We want to disable addition of signature by default. Could you please parametrize this as well? 

@ShipraJain01 @rtanase @ghyadav

Reply:
.., can do it in another PR.